resources/sql_scripts/update_scripts/snmp_version_update.sql

```diff
@@ -0,0 +1,32 @@
+CREATE SEQUENCE public.snmp_versions_snmp_version_id_seq
```

Comment:
Should we add a comment at the top similar to firmware_update.sql to let users know what it's purpose is?

Reply:
I went ahead and added a description for this script. I didn't write this one so I just gave it my best shot based on what the script does.